projects/melon-runtime/MelonJS/Properties/Resources.resx

```diff
@@ -302,7 +302,7 @@ babel/
       "types": [
         "melon-types"
       ],
-      "lib": [ "es2015", "DOM" ]
+      "lib": [ "es2015" ]
```

Comment:
I think it is safe to keep `es2022` instead of `es2015`

Reply:
Awesome